ape-config.yaml

```diff
@@ -32,7 +31,7 @@ ethereum:
     required_confirmations: 1
 
 foundry:
-  port: 8555
+  port: 8545
```

Comment:
I think the default should be 8555 so we don't require people to spin up an an anvil node.  8555 will just spin one up automatically when tests are run.

Reply:
actually, let's add comments here about 8555 and 8545 about which does which (8545 connect to local anvil, 8555 spin up automatically during tests)